src/components/DynamicInputs.tsx

```diff
@@ -1,13 +1,35 @@
 import React, { useCallback } from 'react';
 
+export enum INPUT_LABEL_TYPE {
+  ADDRESS = 'Address',
+  SCOPE = 'Custom',
```

Comment:
```suggestion
  SCOPE = 'Scope',
```

Reply:
resolved in [c2c4d95](https://github.com/MetaMask/test-dapp-multichain/pull/36/commits/c2c4d953cc58698100ed16fe9b2fff2dd83f5c82)